### ISSUE TITLE
How to process docking with two Cu ions?

### ISSUE THREAD

**Issue by pharm-dacnhan** [ccsb-scripps/AutoDock-Vina#205]
We are doing a project related to tyrosine kinase protein, in which we need process the docking step with 2 ions Copper instead of zinc. Thus, we need to modify the file AD4Zn.dat. We added the parameter of Cu (atom_par Cu) with Rii, epsii, vol, and solpar values of 3.50; 0.005; 12.000; and -0.00110. Then, we process Docking. I wonder if this action is correct? We appreciate for any comments from experts. Many thanks!

**Comment by pharm-dacnhan** [comment]
We are doing a project related to tyrosine kinase protein, in which we need process the docking step with 2 ions Copper instead of **Zinc**. Thus, we need to modify the file AD4Zn.dat. We added the parameter of Cu (atom_par Cu) with Rii, epsii, vol, and solpar values of 3.50; 0.005; 12.000; and -0.00110. Then, we process Docking. I wonder if this action is correct? We appreciate for any comments from experts. Many thanks!

**Comment by diogomart** [comment]
Yes, but I'd use a vdW radius more similar to that of the other metals.

**Comment by xavgit** [comment]
Hi,
inspecting the AD4Zn.dat file there are lines for Mg, Ca and other metals different from Zn.
Therefore to make a docking involving a metal present in AD4Zn.dat it is sufficient to follow the 
steps in the tutorial "Docking with zinc metalloproteins"?
I can use the command;
pythonsh <script_directory>/zinc_pseudo.py -r protein.pdbqt -o protein_tz.pdbqt
even in the case of metal that is different from zinc?

Many thanks.
Saverio

**Comment by diogomart** [comment]
Hi,

If there's no Zinc then there's no need to follow the AD4Zn instructions, the regular docking tutorial suffices, because the _AD4Zn.dat_ file does not add any metals that aren't already included in the hard-coded defaults.

**Comment by rwxayheee** [comment]
Closing as resolved. For anyone who's interested in docking with metalloprotein receptors, as of today we're making updates on how receptor and ligand involving metal ions can be prepared which may allow people to perform docking differently. See recent discussion (https://github.com/ccsb-scripps/AutoDock-Vina/issues/368) and feel free to open a new issue with a specific system, that you wish to be able to prepare and dock.
